docs/cmd/kn_service_update.md

```diff
@@ -38,28 +38,31 @@ kn service update NAME [flags]
-  -p, --port int32               The port where application listens on.
-      --requests-cpu string      The requested CPU (e.g., 250m).
-      --requests-memory string   The requested memory (e.g., 64Mi).
-      --revision-name string     The revision name to set. Must start with the service name and a dash as a prefix. Empty revision name will result in the server generating a name for the revision. Accepts golang templates, allowing {{.Service}} for the service name, {{.Generation}} for the generation, and {{.Random [n]}} for n random consonants. (default "{{.Service}}-{{.Random 5}}-{{.Generation}}")
-      --tag strings              Set tag (format: --tag revisionRef=tagName) where revisionRef can be a revision or '@latest' string representing latest ready revision. This flag can be specified multiple times.
-      --traffic strings          Set traffic distribution (format: --traffic revisionRef=percent) where revisionRef can be a revision or a tag or '@latest' string representing latest ready revision. This flag can be given multiple times with percent summing up to 100%.
-      --untag strings            Untag revision (format: --untag tagName). This flag can be spcified multiple times.
-      --wait-timeout int         Seconds to wait before giving up on waiting for service to be ready. (default 60)
+      --async                      Update service and don't wait for it to become ready.
+      --concurrency-limit int      Hard Limit of concurrent requests to be processed by a single replica.
+      --concurrency-target int     Recommendation for when to scale up based on the concurrent number of incoming request. Defaults to --concurrency-limit when given.
+  -e, --env stringArray            Environment variable to set. NAME=value; you may provide this flag any number of times to set multiple environment variables. To unset, specify the environment variable name followed by a "-" (e.g., NAME-).
+      --env-from stringArray       Config a envfrom with a config map or secret. (config-map | secret):CONFIG_MAP_OR_SECRET_NAME you may provide this flag any number of times. To unset, specify the config map name followed by a "-" (e.g., config-map:CONFIG_MAP_NAME- or secret:SECRET_NAME-).
+  -h, --help                       help for update
+      --image string               Image to run.
+  -l, --label stringArray          Service label to set. name=value; you may provide this flag any number of times to set multiple labels. To unset, specify the label name followed by a "-" (e.g., name-).
+      --limits-cpu string          The limits on the requested CPU (e.g., 1000m).
+      --limits-memory string       The limits on the requested memory (e.g., 1024Mi).
+      --lock-to-digest             keep the running image for the service constant when not explicitly specifying the image. (--no-lock-to-digest pulls the image tag afresh with each new revision) (default true)
```

Comment:
Nit ... Start with capital to be consistent with others => "Keep the ..."

Reply:
Similar with previous comment, since `--no-lock-to-digest` is automatically generated, if changing "keep" to "Keep", the description of `--no-lock-to-digest` will be "do not Keep ...".